symplyphysics/laws/chemistry/cross_section_of_interaction_in_model_of_quasirigid_spheres.py

```diff
@@ -16,6 +16,10 @@
 ## g - the cross-sectional area of the interaction of particles,
 ## r - the distance of the greatest convergence of two colliding particles.
 
+# Links: Wikipedia, second formula <https://en.wikipedia.org/wiki/Cross_section_(physics)#Collision_among_gas_particles>
+
+# FIXME: shouldn't it be `(2*r)^2` as per the formula in the link?
```

Comment:
It is probably due to distance between colliding particles = 2 * radius of a particle.

Reply:
You're right, I was confused by the description

Reply:
I'll add a note